src/Xamarin.Android.Build.Tasks/Xamarin.Android.Tooling.targets

```diff
@@ -0,0 +1,72 @@
+<!--
+***********************************************************************************************
+Xamarin.Android.Tooling.targets
+
+This file contains any calls to the <ResolveSdks/> or
+<ResolveJdkJvmPath/> MSBuild tasks. It is imported and used by
+"legacy" Xamarin.Android projects, binding projects, and .NET 5
+projects.
+
+***********************************************************************************************
+-->
+
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <UsingTask TaskName="Xamarin.Android.Tasks.ResolveSdks" AssemblyFile="Xamarin.Android.Build.Tasks.dll" />
+  <UsingTask TaskName="Xamarin.Android.Tasks.ResolveJdkJvmPath" AssemblyFile="Xamarin.Android.Build.Tasks.dll" />
+
+  <PropertyGroup>
+    <_SetLatestTargetFrameworkVersionDependsOnTargets Condition=" '$(_AndroidIsBindingProject)' == 'True'">
+      _ResolveSdks;
+      _ResolveAndroidTooling;
+    </_SetLatestTargetFrameworkVersionDependsOnTargets>
+    <_SetLatestTargetFrameworkVersionDependsOnTargets Condition=" '$(_AndroidIsBindingProject)' != 'True'">
+      _ResolveSdks;
+      _ResolveAndroidTooling;
+      _InjectAaptDependencies;
+      _InjectAapt2Dependencies;
+    </_SetLatestTargetFrameworkVersionDependsOnTargets>
+  </PropertyGroup>
+
+  <Target Name="_SetLatestTargetFrameworkVersion" DependsOnTargets="$(_SetLatestTargetFrameworkVersionDependsOnTargets)">
+    <PropertyGroup>
+      <TargetFrameworkMoniker Condition=" '$(TargetFrameworkProfile)' == '' ">$(TargetFrameworkIdentifier),Version=$(TargetFrameworkVersion)</TargetFrameworkMoniker>
+      <TargetFrameworkMoniker Condition=" '$(TargetFrameworkProfile)' != '' ">$(TargetFrameworkIdentifier),Version=$(TargetFrameworkVersion),Profile=$(TargetFrameworkProfile)</TargetFrameworkMoniker>
+      <NuGetTargetMoniker>$(TargetFrameworkMoniker)</NuGetTargetMoniker>
+      <TargetFrameworkMonikerAssemblyAttributesPath>$(IntermediateOutputPath)$(TargetFrameworkMoniker).AssemblyAttributes$(DefaultLanguageSourceExtension)</TargetFrameworkMonikerAssemblyAttributesPath>
+    </PropertyGroup>
+    <ItemGroup>
+      <FileWrites Include="$(TargetFrameworkMonikerAssemblyAttributesPath)" />
+      <Reference Include="$(_JavaInteropReferences)" />
```

Comment:
I think this can use a `Condition=" '$(UsingAndroidNETSdk)' != 'True' "` but we can address this case in a future PR that is dealing with ResolveAssemblies changes.